### YamlMime:UniversalReference
ms.openlocfilehash: ea79a2eb5c1197061788aa004ade361c3c84c160
ms.sourcegitcommit: 286f52ea38c9eff2ec9d4f8cabeb86f62fd9c406
ms.translationtype: MT
ms.contentlocale: it-IT
ms.lasthandoff: 08/22/2018
ms.locfileid: "41683460"
items:
- uid: azure-event-hubs.ErrorNameConditionMapper
  name: ErrorNameConditionMapper
  fullName: ErrorNameConditionMapper
  children:
  - azure-event-hubs.ErrorNameConditionMapper.ArgumentError
  - azure-event-hubs.ErrorNameConditionMapper.UnauthorizedError
  - azure-event-hubs.ErrorNameConditionMapper.ConnectionForcedError
  - azure-event-hubs.ErrorNameConditionMapper.ConnectionRedirectError
  - azure-event-hubs.ErrorNameConditionMapper.DecodeError
  - azure-event-hubs.ErrorNameConditionMapper.DetachForcedError
  - azure-event-hubs.ErrorNameConditionMapper.ErrantLinkError
  - azure-event-hubs.ErrorNameConditionMapper.FrameSizeTooSmallError
  - azure-event-hubs.ErrorNameConditionMapper.FramingError
  - azure-event-hubs.ErrorNameConditionMapper.HanldeInUseError
  - azure-event-hubs.ErrorNameConditionMapper.IllegalStateError
  - azure-event-hubs.ErrorNameConditionMapper.InternalServerError
  - azure-event-hubs.ErrorNameConditionMapper.InvalidFieldError
  - azure-event-hubs.ErrorNameConditionMapper.InvalidOperationError
  - azure-event-hubs.ErrorNameConditionMapper.LinkRedirectError
  - azure-event-hubs.ErrorNameConditionMapper.MessageLockLostError
  - azure-event-hubs.ErrorNameConditionMapper.MessageTooLargeError
  - azure-event-hubs.ErrorNameConditionMapper.MessagingEntityDisabledError
  - azure-event-hubs.ErrorNameConditionMapper.ArgumentOutOfRangeError
  - azure-event-hubs.ErrorNameConditionMapper.OperationCancelledError
  - azure-event-hubs.ErrorNameConditionMapper.PreconditionFailedError
  - azure-event-hubs.ErrorNameConditionMapper.QuotaExceededError
  - azure-event-hubs.ErrorNameConditionMapper.ReceiverDisconnectedError
  - azure-event-hubs.ErrorNameConditionMapper.ResourceDeletedError
  - azure-event-hubs.ErrorNameConditionMapper.ResourceLockedError
  - azure-event-hubs.ErrorNameConditionMapper.SenderBusyError
  - azure-event-hubs.ErrorNameConditionMapper.ServerBusyError
  - azure-event-hubs.ErrorNameConditionMapper.ServiceCommunicationError
  - azure-event-hubs.ErrorNameConditionMapper.ServiceUnavailableError
  - azure-event-hubs.ErrorNameConditionMapper.SessionCannotBeLockedError
  - azure-event-hubs.ErrorNameConditionMapper.SessionLockLostError
  - azure-event-hubs.ErrorNameConditionMapper.SessionWindowViolationError
  - azure-event-hubs.ErrorNameConditionMapper.SystemError
  - azure-event-hubs.ErrorNameConditionMapper.TransferLimitExceededError
  - azure-event-hubs.ErrorNameConditionMapper.UnattachedHandleError
  - azure-event-hubs.ErrorNameConditionMapper.NotImplementedError
  langs:
  - typeScript
  type: enum
  summary: <span data-ttu-id="d1e62-101">I nomi di errore viene eseguito il mapping per le condizioni di errore di amqp.</span><span class="sxs-lookup"><span data-stu-id="d1e62-101">Maps the Error names to the amqp error conditions.</span></span>
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ArgumentError
  name: ArgumentError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-102">Errore viene generato quando è stato ricevuto un argomento non corretto.</span><span class="sxs-lookup"><span data-stu-id="d1e62-102">Error is thrown when an incorrect argument was received.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.UnauthorizedError
  name: UnauthorizedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-103">Quando i parametri di connessione sono errati e il server ha rifiutato la connessione, viene generato l'errore.</span><span class="sxs-lookup"><span data-stu-id="d1e62-103">Error is thrown when the connection parameters are wrong and the server refused the connection.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ConnectionForcedError
  name: ConnectionForcedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-104">Quando un operatore intervenivano per decidere per chiudere la connessione per qualche motivo, viene generato l'errore.</span><span class="sxs-lookup"><span data-stu-id="d1e62-104">Error is thrown when an operator intervened to close the connection for some reason.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ConnectionRedirectError
  name: ConnectionRedirectError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-105">Errore viene generato quando il contenitore non sono più disponibile nella connessione corrente.</span><span class="sxs-lookup"><span data-stu-id="d1e62-105">Error is thrown when the container is no longer available on the current connection.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.DecodeError
  name: DecodeError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-106">Errore viene generato quando non è stato possibile decodificare i dati.</span><span class="sxs-lookup"><span data-stu-id="d1e62-106">Error is thrown when data could not be decoded.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.DetachForcedError
  name: DetachForcedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-107">Quando un operatore intervenivano per decidere per scollegare per qualche motivo, viene generato l'errore.</span><span class="sxs-lookup"><span data-stu-id="d1e62-107">Error is thrown when an operator intervened to detach for some reason.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ErrantLinkError
  name: ErrantLinkError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-108">Errore viene generato quando è stato ricevuto l'input per un collegamento che è stato scollegato con un errore.</span><span class="sxs-lookup"><span data-stu-id="d1e62-108">Error is thrown when input was received for a link that was detached with an error.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.FrameSizeTooSmallError
  name: FrameSizeTooSmallError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-109">Errore viene generato quando il peer non è possibile inviare un frame perché la codifica più piccolo della performativa con i valori attualmente validi sarebbe troppo grande per adattarlo all'interno di un frame della dimensione massima di frame concordata.</span><span class="sxs-lookup"><span data-stu-id="d1e62-109">Error is thrown when the peer cannot send a frame because the smallest encoding of the performative with the currently valid values would be too large to fit within a frame of the agreed maximum frame size.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.FramingError
  name: FramingError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-110">Errore viene generato quando un'intestazione di intervallo valido non può essere formata da flusso di byte in ingresso.</span><span class="sxs-lookup"><span data-stu-id="d1e62-110">Error is thrown when a valid frame header cannot be formed from the incoming byte stream.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.HanldeInUseError
  name: HanldeInUseError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-111">Errore viene generato quando un'operazione di connessione è stato ricevuto tramite un handle che è già in uso per un collegamento associato.</span><span class="sxs-lookup"><span data-stu-id="d1e62-111">Error is thrown when an attach was received using a handle that is already in use for an attached link.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.IllegalStateError
  name: IllegalStateError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-112">Errore viene generato quando il peer ha inviato un frame che non è consentito nello stato corrente.</span><span class="sxs-lookup"><span data-stu-id="d1e62-112">Error is thrown when the peer sent a frame that is not permitted in the current state.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.InternalServerError
  name: InternalServerError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-113">Quando si è verificato un errore interno del server, viene generato l'errore.</span><span class="sxs-lookup"><span data-stu-id="d1e62-113">Error is thrown when an internal server error occured.</span></span> <span data-ttu-id="d1e62-114">Si hanno trovato un bug?</span><span class="sxs-lookup"><span data-stu-id="d1e62-114">You may have found a bug?</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.InvalidFieldError
  name: InvalidFieldError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-115">Errore viene generato quando un campo non valido passato nel corpo di un frame e l'operazione non è possibile procedere.</span><span class="sxs-lookup"><span data-stu-id="d1e62-115">Error is thrown when an invalid field was passed in a frame body, and the operation could not proceed.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.InvalidOperationError
  name: InvalidOperationError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-116">Quando viene tentata un'operazione, ma non è consentita, viene generato l'errore.</span><span class="sxs-lookup"><span data-stu-id="d1e62-116">Error is thrown when an operation is attempted but is not allowed.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.LinkRedirectError
  name: LinkRedirectError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-117">Errore viene generato quando l'indirizzo fornito non può essere risolta in una parte finale nel contenitore corrente.</span><span class="sxs-lookup"><span data-stu-id="d1e62-117">Error is thrown when the address provided cannot be resolved to a terminus at the current container.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.MessageLockLostError
  name: MessageLockLostError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-118">Errore viene generato quando il blocco sul messaggio viene perso.</span><span class="sxs-lookup"><span data-stu-id="d1e62-118">Error is thrown when the lock on the message is lost.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.MessageTooLargeError
  name: MessageTooLargeError
  children: []
  langs:
  - typeScript
  summary: '<span data-ttu-id="d1e62-119">Errore viene generato quando il messaggio inviato è troppo grande: la dimensione massima è 256Kb.</span><span class="sxs-lookup"><span data-stu-id="d1e62-119">Error is thrown when the message sent is too large: the maximum size is 256Kb.</span></span>'
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.MessagingEntityDisabledError
  name: MessagingEntityDisabledError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-120">Quando si tenta di accedere o connettersi a un'entità di messaggistica disabilitata, viene generato l'errore.</span><span class="sxs-lookup"><span data-stu-id="d1e62-120">Error is thrown when trying to access/connect to a disabled messaging entity.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ArgumentOutOfRangeError
  name: ArgumentOutOfRangeError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-121">Errore viene generato quando un argomento ha un valore non compreso nell'intervallo accettabile.</span><span class="sxs-lookup"><span data-stu-id="d1e62-121">Error is thrown when an argument has a value that is out of the admissible range.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.OperationCancelledError
  name: OperationCancelledError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-122">Errore viene generato quando server Annulla l'operazione a causa di un problema interno.</span><span class="sxs-lookup"><span data-stu-id="d1e62-122">Error is thrown when server cancels the operation due to an internal issue.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.PreconditionFailedError
  name: PreconditionFailedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-123">Errore viene generato quando una condizione che deve rispettare per poter eseguire un'operazione non è stato.</span><span class="sxs-lookup"><span data-stu-id="d1e62-123">Error is thrown when a condition that should have been met in order to execute an operation was not.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.QuotaExceededError
  name: QuotaExceededError
  children: []
  langs:
  - typeScript
  summary: >-
    <span data-ttu-id="d1e62-124">Errore viene generato il è stata superata la quota di Azure hub eventi/bus di servizio.</span><span class="sxs-lookup"><span data-stu-id="d1e62-124">Error is thrown the the Azure EventHub/ServiceBus quota has been exceeded.</span></span>

    <span data-ttu-id="d1e62-125">Le quote vengono reimpostate periodicamente, questa operazione dovrà attendere fino a quel momento.</span><span class="sxs-lookup"><span data-stu-id="d1e62-125">Quotas are reset periodically, this operation will have to wait until then.</span></span>

    <span data-ttu-id="d1e62-126">L'entità di messaggistica ha raggiunto le dimensioni massime consentite.</span><span class="sxs-lookup"><span data-stu-id="d1e62-126">The messaging entity has reached its maximum allowable size.</span></span>

    <span data-ttu-id="d1e62-127">Questa situazione può verificarsi se a livello di gruppo di consumer è già stato aperto il numero massimo di ricevitori, ovvero 5.</span><span class="sxs-lookup"><span data-stu-id="d1e62-127">This can happen if the maximum number of receivers (which is 5) has already been opened on a per-consumer group level.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ReceiverDisconnectedError
  name: ReceiverDisconnectedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-128">Errore viene generato quando due o più istanze si connettono alla stessa partizione con i valori diversi dal valore epoch.</span><span class="sxs-lookup"><span data-stu-id="d1e62-128">Error is thrown when two or more instances connect to the same partition with different epoch values.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ResourceDeletedError
  name: ResourceDeletedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-129">Errore viene generato quando un'entità server che collabora con il client è stata eliminata.</span><span class="sxs-lookup"><span data-stu-id="d1e62-129">Error is thrown when a server entity the client is working with has been deleted.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ResourceLockedError
  name: ResourceLockedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-130">Errore viene generato quando il client ha tentato di usare un'entità server a cui non ha accesso perché un altro client sta usando.</span><span class="sxs-lookup"><span data-stu-id="d1e62-130">Error is thrown when the client attempted to work with a server entity to which it has no access because another client is working with it.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.SenderBusyError
  name: SenderBusyError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-131">Errore viene generato quando il mittente di client non dispone di sufficienti crediti di collegamento per inviare il messaggio.</span><span class="sxs-lookup"><span data-stu-id="d1e62-131">Error is thrown when the client sender does not have enough link credits to send the message.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ServerBusyError
  name: ServerBusyError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-132">Quando il server è occupato, viene generato l'errore.</span><span class="sxs-lookup"><span data-stu-id="d1e62-132">Error is thrown when the server is busy.</span></span> <span data-ttu-id="d1e62-133">I chiamanti devono attendere qualche minuto e ripetere l'operazione.</span><span class="sxs-lookup"><span data-stu-id="d1e62-133">Callers should wait a while and retry the operation.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ServiceCommunicationError
  name: ServiceCommunicationError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-134">Errore per la segnalazione degli errori di comunicazione generale relativi alle operazioni di messaggistica.</span><span class="sxs-lookup"><span data-stu-id="d1e62-134">Error for signaling general communication errors related to messaging operations.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ServiceUnavailableError
  name: ServiceUnavailableError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-135">Quando il servizio non è disponibile, viene generato l'errore.</span><span class="sxs-lookup"><span data-stu-id="d1e62-135">Error is thrown when the service is unavailable.</span></span> <span data-ttu-id="d1e62-136">L'operazione deve essere ritentata.</span><span class="sxs-lookup"><span data-stu-id="d1e62-136">The operation should be retried.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.SessionCannotBeLockedError
  name: SessionCannotBeLockedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-137">Quando la sessione del bus di servizio di Azure non può essere bloccata, viene generato l'errore.</span><span class="sxs-lookup"><span data-stu-id="d1e62-137">Error is thrown when the Azure ServiceBus session cannot be locked.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.SessionLockLostError
  name: SessionLockLostError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-138">Errore viene generato quando il blocco sulla sessione del bus di servizio di Azure viene perso.</span><span class="sxs-lookup"><span data-stu-id="d1e62-138">Error is thrown when the lock on the Azure ServiceBus session is lost.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.SessionWindowViolationError
  name: SessionWindowViolationError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-139">Errore viene generato quando il peer ha violato finestra in ingresso per la sessione.</span><span class="sxs-lookup"><span data-stu-id="d1e62-139">Error is thrown when the peer violated incoming window for the session.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.SystemError
  name: SystemError
  children: []
  langs:
  - typeScript
  summary: >-
    <span data-ttu-id="d1e62-140">Errore viene generato quando viene generato un errore di sistema di livello basso da Node. js.</span><span class="sxs-lookup"><span data-stu-id="d1e62-140">Error is thrown when a low level system error is thrown by node.js.</span></span>

    <xref:https%3A%2F%2Fnodejs.org%2Fapi%2Ferrors.html%23errors_class_systemerror>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.TransferLimitExceededError
  name: TransferLimitExceededError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-141">Errore viene generato quando il peer ha inviato più i trasferimenti di messaggi superiore a quello attualmente consentito il collegamento.</span><span class="sxs-lookup"><span data-stu-id="d1e62-141">Error is thrown when the peer sent more message transfers than currently allowed on the link.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.UnattachedHandleError
  name: UnattachedHandleError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-142">Quando è stato ricevuto un frame (diversi da collegare a) che fanno riferimento a un handle che non è attualmente in uso di un collegamento, viene generato l'errore.</span><span class="sxs-lookup"><span data-stu-id="d1e62-142">Error is thrown when a frame (other than attach) was received referencing a handle which is not currently in use of an attached link.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.NotImplementedError
  name: NotImplementedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="d1e62-143">Errore viene generato quando una funzionalità non ancora implementata ma è presente il segnaposto.</span><span class="sxs-lookup"><span data-stu-id="d1e62-143">Error is thrown when a feature is not implemented yet but the placeholder is present.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
